account_bank_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -106,6 +106,7 @@ class OnlineBankStatementProvider(models.Model):
     certificate_public_key = fields.Text()
     certificate_private_key = fields.Text()
     certificate_chain = fields.Text()
+    allow_empty_statements = fields.Boolean(string="Allow empty statements")
```

Comment:
`string` is not needed, it will be automatically named like that. Also, you're changing default behavior - the default value should be `True` to keep things unaltered. Also, explaining that in README would be great

Reply:
But I stand corrected about the default value due to this https://github.com/OCA/bank-statement-import/pull/408#issuecomment-902509072

Reply:
I deleted the string